.github/workflows/public-api-check.yml

```diff
@@ -0,0 +1,31 @@
+name: Public API check
+
+on:
+  pull_request:
+
+jobs:
+  commentPR:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: mshick/add-pr-comment@v1
```

Comment:
Just curious, what's our policy on using actions that are not from the standard `actions`?

Reply:
Not using non-github actions now :+1: